pkg/kube/vault/vault_factory.go

```diff
@@ -167,6 +169,7 @@ func waitForVault(vaultClient *api.Client, initialDelay, timeout time.Duration)
 func waitForKVEngine(vaultClient *api.Client, initialDelay, timeout time.Duration) error {
 	return util.RetryWithInitialDelaySlower(initialDelay, timeout, func() error {
 		if _, err := vaultClient.Logical().Read(kvEngineConfigPath); err != nil {
+			log.Logger().Info("Waiting for KV engine to be configured...")
```

Comment:
This doesn't seem right - it's just before returning an error

Reply:
The retry takes place on error. This log is intended to show some action when a new retry is being performed.

Reply:
oic